code/obj/item/device/scanners.dm

```diff
@@ -232,6 +237,21 @@ that cannot be itched
 				icon_state = "fs_pinfar"
 		SPAWN(0.5 SECONDS)
 			.(T)
+	verb/set_phrase()
+		set name = "Print last report"
+		set category = "Local"
+
+		if (!src.printing )
+			src.printing = 1
+			playsound(src, "sound/machines/printer_thermal.ogg", 50, 1)
+			SPAWN(1 SECONDS)
+				var/obj/item/paper/P = new /obj/item/paper
+				P.set_loc(get_turf(src))
+
+				P.info = last_scan
+				P.name = "Forensic readout"
+			src.printing = null
```

Comment:
Instead of a `printing` var I'd rather you do something like `if(!ON_COOLDOWN(src, "print", 1 SECOND))`